acoustid.py

```diff
@@ -265,9 +265,11 @@ def parse_lookup_result(data):
 
         for recording in result['recordings']:
             # Get the artist if available.
-            if recording.get('artists'):
-                names = [artist['name'] for artist in recording['artists']]
-                artist_name = '; '.join(names)
+            artists = recording.get("artists")
+            if artists:
+                artist_name = ""
+                for artist in artists:
+                    artist_name += artist["name"] + artist.get("joinphrase", "")
```

Comment:
This might be slightly nicer to write with a list comprehension and `.join`?

```py
[
    artist["name"] + artist.get("joinphrase", "")
    for artist in artists
].join("")
```